app/serializers/organization_serializer.rb

```diff
@@ -1,5 +1,4 @@
 class OrganizationSerializer < ActiveModel::Serializer
-  attributes :name, :slug, :url
+  attributes :name, :website, :kind
```

Comment:
Looking through the project, it appears that there is no other place that `OrganizationSerializer` is used. Which means that this change is totally great and makes perfect sense! Is that what you found too?

Reply:
Ya I don't think it was being used 👍